### PR TITLE
Add support git_subdir resource spec

### PR DESCRIPTION
**Pull request by kraysx7**

### CODE REVIEW

File: src/rebar_raw_resource.erl [hunk 1]
```diff
@@ -426,9 +435,10 @@ make_vsn(Path) ->
...
 make_vsn(AppInfo0, _ResourceState) ->
     Spec = case rebar_app_info:source(AppInfo0) of
-        {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}} -> {Res0, Loc0, Ref};
-        {?RTYPE, S, _} -> S;
-        {?RTYPE, S} -> S
+               {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}, SubLoc} -> {Res0, Loc0, Ref, SubLoc};
+               {?RTYPE, Loc0, #mod_ref{res = Res0, ref = Ref}} -> {Res0, Loc0, Ref};
+               {?RTYPE, S, _} -> S;
+               {?RTYPE, S} -> S
     end,
```
Comment (nalundgaard) [inline src/rebar_raw_resource.erl]: As long as you're re-indenting to align with the `case` here, shouldn't the `end` be indented to line up with the `case` as well?

**Review by nalundgaard — CHANGES_REQUESTED** [review]
Thanks for the contribution! I hadn't seen git_subdir in use yet. Have you used this in a project yet? I am curious if you've verified it and if so, how.

**Comment by kraysx7** [comment]
> Thanks for the contribution! I hadn't seen git_subdir in use yet. Have you used this in a project yet? I am curious if you've verified it and if so, how.

Yes, I checked how the plugin works in a working project. I needed to pull protobuf files from another repository from the subdirectory and build them using the gpb plugin

File: src/rebar_raw_resource.erl (hunk 1 above)
Comment (kraysx7) [reply src/rebar_raw_resource.erl]: emacs aligned the code this way...

**Review by nalundgaard — CHANGES_REQUESTED** [review]
What about [`download/4`](https://github.com/alertlogic/rebar_raw_resource/pull/5/files#diff-66a289679bb22cbba6c17271aa075becR296-R297)? It looks like there is a subdir case not handled there.

**Review by nalundgaard — APPROVED** [review]

**Comment by kraysx7** [comment]
Do you think about making a new tag (maybe 0.10.0 or 1.0.0) and publishing it in hex pm?

**Comment by nalundgaard** [comment]
Yes, for sure.

**Comment by kraysx7** [comment]
> Yes, for sure.

Thank you for your quick response! Good luck!

**Comment by nalundgaard** [comment]
> Thank you for your quick response! Good luck!

Published 0.10.0 and added #6 to remove the maintainers property that is no longer permitted by Hex packager. Thanks again!

**Pull request merged**